src/main/java/org/opentripplanner/model/Pathway.java

```diff
@@ -7,9 +7,21 @@
 
     private static final int MISSING_VALUE = -999;
 
+    private static final int DEFAULT_TRAVERSAL_TIME = 5;
+
+    private static final int DEFAULT_WHEELCHAIR_TRAVERSAL_TIME = 10;
+
+    private static final float WALK_SPEED_FEET_SECONDS = 4.59f;
```

Comment:
The data that come from pathways.txt are in meters, not feet, so these numbers here have to be converted to be appropriate for meters.

Reply:
Fixed, thanks.